src/features/communication/communication.tsx

```diff
@@ -16,23 +21,29 @@ const localizedContent = {
     subTitle: 'Discord is the main communication channel in RS School',
     firstParagraphFirstHalf: 'Here is link for the',
     discordLink: 'course discord server',
-    firstParagraphSecondHalf: 'where you can see latest news and chat with students.',
-    secondParagraphFirstHalf: 'More info about communication at RS School in the',
-    rsDocsLink: 'RS Docs',
+    firstParagraphSecondHalf: ' where you can see latest news and chat with students.',
+    secondParagraphFirstHalf: 'There are channels in',
+    telegramLink: 'Telegram',
     secondParagraphSecondHalf:
-      'Here you could find rules, links to other courses and Telegram channels related to your country or even city. Feel free to use it :)',
+      ', for discussing events related to your location. For example, offline lectures. And informal chats among students from the same location.',
+    thirdParagraphFirstHalf: 'Please read the information about communication in RS School in the',
+    rsDocsLink: 'RS Docs',
+    thirdParagraphSecondHalf: ', where you can find rules, descriptions of channels, FAQ.',
   },
   ru: {
     title: 'Общение',
     subTitle: 'Дискорд — основной способ общения в RS School',
     firstParagraphFirstHalf: 'Вот ссылка на',
     discordLink: 'дискорд сервер курса',
     firstParagraphSecondHalf:
-      ', где вы можете посмотреть последние новости и общаться со студентами.',
-    secondParagraphFirstHalf: 'Более подробная информация об общении в RS School в',
-    rsDocsLink: 'RS Docs',
+      ', где вы можете посмотреть последние новости, задать вопросы и общаться со студентами.',
+    secondParagraphFirstHalf: 'Также есть каналы в',
+    telegramLink: 'телеграм',
```

Comment:
Why is it capitalized T in the English version, and lowercase in the Russian version?